dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/jobs/steps/processors/AssemblyCheckFilterProcessor.java

```diff
@@ -45,7 +45,14 @@ public AssemblyCheckFilterProcessor(FastaSequenceReader fastaSequenceReader) {
      */
     @Override
     public SubSnpCoreFields process(SubSnpCoreFields subSnpCoreFields) throws Exception {
-        String referenceAllele = subSnpCoreFields.getReferenceInForwardStrand();
+        String referenceAllele;
+        try {
+            referenceAllele = subSnpCoreFields.getReferenceInForwardStrand();
+        } catch (IllegalArgumentException hgvsReferenceUndefined) {
```

Comment:
Having to name the variable in some way to understand what is going on makes me think that we need a specific exception type here...